consensus/src/consensus/mod.rs

```diff
@@ -125,6 +160,24 @@ impl<N: Network> Consensus<N> {
             head_requests_time: None,
             min_peers,
             zkp_proxy,
+            remote_events: remote_tx,
+        }
+    }
+
+    fn init_remote_event_dispatcher(network: &Arc<N>, blockchain: &BlockchainProxy) {
+        // We spawn the Remote Event Dispatcher into its own task (this is only available for full nodes and history nodes)
+
+        match blockchain {
+            #[cfg(feature = "full")]
+            BlockchainProxy::Full(blockchain) => {
```

Comment:
Do you need the `cfg` switch here? It's already matching against the blockchain type.

Reply:
Yes, because the "`BlockchainProxy::Full`" type is not defined in the web client.